src/AzureAuth/CommandMain.cs

```diff
@@ -347,9 +347,35 @@ public int OnExecute()
             // Small bug in Lasso - Add does not accept a null IEnumerable here.
             this.eventData.Add("settings_scopes", this.authSettings.Scopes ?? new List<string>());
 
+            if (this.InteractivityDisabled())
+            {
+                this.eventData.Add("interactivity_disabled", true);
+                this.logger.LogCritical("Interactive authentication is disabled.");
+
+                return 1;
+            }
+
             return this.ClearCache ? this.ClearLocalCache() : this.GetToken();
         }
 
+        /// <summary>
+        /// Determines whether the given env is user interactive or not.
+        /// </summary>
+        /// <returns>A boolean to indicate user interactive env</returns>
+        public bool InteractivityDisabled()

```

Comment:
```suggestion
        public bool PCADisabled()
```
I would use the langauge PCA internally since that's what we really mean. Offline conversation about naming here, switching to "AZUERAUTH_NO_USER"